protos/generated/message_pb2.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
```

Comment:
a little surprised that this file is coming out as `_pb2` when the proto file has `syntax = "proto3"` in it 🤔 any idea why?

Reply:
Yeah I was also surprised the first time I generated the proto files but it seems like `syntax = "proto3"` has nothing to do with `_pb2` 
https://github.com/grpc/grpc/issues/15444#issuecomment-396442980

Reply:
thanks for the link, makes sense!